algorithms/maths/diffie_hellman_key_exchange.py

```diff
@@ -1,5 +1,5 @@
 import math
-from random import randint
+from random import _randint
```

Comment:
Why is this line change required?

Reply:
no. just a small improvement

Reply:
It isnt required but is a minor improvement as it shows that the randint function is not to be used whilest using the module. It shows it is private.

Reply:
Please validate it unit tests are passing. 
Refer Tests here: https://github.com/keon/algorithms